packages/engine/src/scene/systems/SceneObjectDynamicLoadSystem.ts

```diff
@@ -63,7 +82,34 @@ export default async function SceneObjectDynamicLoadSystem(world: World) {
             position: data.position
           })
 
+          hasComponent(entity, RigidBodyComponent) && Physics.removeCollidersFromRigidBody(entity, world.physicsWorld)
+          const targetNode = nodeMap.get(entity)
+          if (targetNode) {
+            iterateEntityNode(targetNode, (node) => {
+              node !== targetNode &&
+                matchActionOnce(
+                  SceneDynamicLoadAction.load.matches.validate((action) => action.uuid === targetNode.uuid, ''),
+                  () => {
+                    const data = { name: node.uuid, components: serializeEntity(node.entity) }
+                    if (node.parentEntity) {
+                      const parentNode = nodeMap.get(node.parentEntity!)!
+                      data['parent'] = parentNode.uuid
+                    }
+                    createSceneEntity(node.uuid, data)
+                  }
+                )
+
+              hasComponent(node.entity, RigidBodyComponent) &&
```

Comment:
once again, should be done in physics system

Reply:
now in physics system